### ISSUE TITLE
Cloning OVPN Configs failing

### ISSUE THREAD

**Issue by Choff3** [haugene/docker-transmission-openvpn#2617]
### Is there a pinned issue for this?

- [X] I have read the pinned issues and could not find my issue

### Is there an existing or similar issue/discussion for this?

- [X] I have searched the existing issues
- [X] I have searched the existing discussions

### Is there any comment in the documentation for this?

- [X] I have read the documentation, especially the FAQ and Troubleshooting parts

### Is this related to a provider?

- [X] I have checked the provider repo for issues
- [X] My issue is NOT related to a provider

### Are you using the latest release?

- [X] I am using the latest release

### Have you tried using the dev branch latest?

- [X] I have tried using dev branch

### Docker run config used

transmission-openvpn:
        image: haugene/transmission-openvpn:dev
        container_name: transmission-openvpn
        restart: unless-stopped
        cap_add:
            - NET_ADMIN
        volumes:
            - '/media/Movies:/movies'
            - '/media/TVShows:/tvshows'
            - '/media/Music:/music'
            - '/transmission:/config'
            - '/storage/transmission:/config'
        environment:
            - PUID=1000
            - PGID=1000
            - OPENVPN_PROVIDER=MULLVAD
            - OPENVPN_CONFIG=us_atl
            - OPENVPN_USERNAME=[removed]
            - OPENVPN_PASSWORD=[removed]
            - LOCAL_NETWORK=[removed]/16
            - TRANSMISSION_WEB_UI=transmission-web-control
            - WEBPROXY_ENABLED=false
            - TRANSMISSION_UMASK=0
        logging:
            driver: json-file
            options:
                max-size: 5m
        sysctls:
            net.ipv6.conf.all.disable_ipv6: 0 #For Mullvad
        ports:
            - '9091:9091'

### Current Behavior

When the container runs the fetch-external-configs script during startup, it fails to clone the `haugene/vpn-configs-contrib` repo. The script tries to clone using branch "main}" which does not exist. 

### Expected Behavior

The fetch-external-configs script should clone the `haugene/vpn-configs-contrib` repo using branch "main" instead of branch "main}".

### How have you tried to solve the problem?

Looked for issues here and on the `haugene/vpn-configs-contrib` repo. Looked at logs to figure out why the container wasn't starting.

### Log output

```
Starting container with revision: 1644c9803c68c6db74851de18c00bb579e1c639b

TRANSMISSION_HOME is currently set to: /config/transmission-home

Creating TUN device /dev/net/tun

Using OpenVPN provider: MULLVAD

Running with VPN_CONFIG_SOURCE auto

No bundled config script found for MULLVAD. Defaulting to external config

Will get configs from https://github.com/haugene/vpn-configs-contrib.git

Cloning https://github.com/haugene/vpn-configs-contrib.git into /config/vpn-configs-contrib

Cloning into '/config/vpn-configs-contrib'...

fatal: Remote branch main} not found in upstream origin
```

### HW/SW Environment

```markdown
- OS: Debian 11
- Docker: Docker version 20.10.5+dfsg1, build 55c4c88
```

### Anything else?

The above log output repeats after the last line.

**Comment by Choff3** [comment]
Switching from `dev` back to `latest` seems to have fixed my issue.

**Comment by pkishino** [comment]
fixed in #42eb2ee
